app/views/declaration/summary/commodity_measure_section.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.DeclarationType._
+@import models.declaration.ExportItem
+@import forms.declaration.CommodityMeasure
+@import models.requests.JourneyRequest
+@import models.viewmodels.HtmlTableRow
+@import models.Mode
+
+@(mode: Mode, item: ExportItem, commodityMeasure: CommodityMeasure)(implicit messages: Messages, journeyRequest: JourneyRequest[_])
+
+@if(!journeyRequest.isType(DeclarationType.CLEARANCE)) {
```

Comment:
I know it will be longer but probably it can be more readable without negation. So specify the list off allowed journeys instead of not allowed.

Reply:
Won't that be less maintainable if/when we add another dec type?   I could create a `isNotType` method on JourneyRequest though.